graphql2/graphqlapp/escalationpolicy.go

```diff
@@ -246,6 +247,9 @@ func (m *Mutation) UpdateEscalationPolicyStep(ctx context.Context, input graphql
 		if input.Targets != nil {
 			step.Targets = make([]assignment.Target, len(input.Targets))
 			for i, tgt := range input.Targets {
+				if tgt.Type == assignment.TargetTypeChanWebhook && !cfg.ValidWebhookURL(tgt.ID) {
+					return validation.NewFieldError("targets", "URL not allowed by administrator")
```

Comment:
Use `[%d]` to denote the error within a list
```suggestion
					return validation.NewFieldError(fmt.Sprintf("Targets[%d]", i), "URL not allowed by administrator")
```

Reply:
I tried this and it resulted in the error not displaying properly on the form. Tried lowercase `targets` as well.

Reply:
ah, I see, if that already works properly, and is being used elsewhere, would you mind just adding a comment explaining the _why_? Something like `the UI code expects targets to be un-indexed` should suffice.

Thank you for the context and for digging into it.

Reply:
no problem! just added the comment to both instances of the code